src/pages/trading-specification/components/_elements.tsx

```diff
@@ -235,18 +235,18 @@ export const TableCellGroup = ({ data, market }: TTableCellGroup) => {
     const { symbol, instrument: text, dl_icon, swf_icon } = data
     const [show_popUp, setShowPopUp] = useState(false)
     const [popup_type, setPopupType] = useState<TPopupType>()
-    const [scrollY, setScrollY] = useState<number>()
+    const [scroll_y, setScroll_y] = useState(0)
```

Comment:
```suggestion
    const [scroll_y, setScrollY] = useState(0)
```

functions are kept camelcase